pkg/auth/dependency/sso/authurl.go

```diff
@@ -30,6 +30,12 @@ func RedirectURIForAPI(urlPrefix *url.URL, providerConfig config.OAuthProviderCo
 	return u.String()
 }
 
+func RedirectURIForWebApp(urlPrefix *url.URL, providerConfig config.OAuthProviderConfiguration) string {
+	u := *urlPrefix
+	u.Path = path.Join(u.Path, fmt.Sprintf("sso/%s/callback", url.PathEscape(providerConfig.ID)))
```

Comment:
I think `/oauth2` should be used by the endpoints when auth gear is being the oidc provider? Since here auth gear is the rp, so the paths should be different? Open to discuss.

Reply:
What about `sso/oauth2/callback/%s`?

Reply:
Looks good to me!